### DISCUSSION TITLE
Add organizationId Parameter to getOrganizationList in OrganizationApi

### DISCUSSION THREAD

**Discussion by mario-jerkovic** [clerk/javascript#3153] [Feature Requests]
### Hello 👋

The `getOrganizationList` method in the `OrganizationApi` could greatly benefit from the ability to fetch organizations by their IDs, similar to how `getUserList` in the `UserApi` allows fetching users by their `userId?: string[]`. This feature would enhance the API's functionality and provide consistency across the API suite.

### Expected Behavior
The `getOrganizationList` function in the `OrganizationApi.ts` should support an `organizationId: string[]` parameter, allowing developers to fetch multiple specific organizations in a single API call based on their IDs.

### Current Behavior
Currently, the `getOrganizationList` method does not allow fetching organizations based on an array of organization IDs. The method signature is as follows:

```typescript
type GetOrganizationListParams = ClerkPaginationRequest<{
    includeMembersCount?: boolean;
    organizationId?: string[]; // <--- New Parameter
    query?: string;
}>

class OrganizationAPI extends AbstractAPI {
    getOrganizationList(params?: GetOrganizationListParams): Promise<PaginatedResourceResponse<Organization[]>>;
}
```

**Comment by jescalan** [comment]
https://github.com/orgs/clerk/discussions/3829
